sct_docker.py

```diff
@@ -68,7 +68,11 @@ def generate(distro="debian:7", version="3.1.1", commands=None, name=None,
 		frag += "\n" + """
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 RUN apt-get update
-RUN apt-get install -y curl
+#Freeglut is needed to avoid warning generated by the use of the plugin in Fsleyes
+RUN apt-get install -y curl vim  freeglut3-dev --fix-missing
```

Comment:
to follow the logic the script, why not putting it under the `if install_fsleyes` relevant cases (i.e. linux distrox), as done [here](https://github.com/neuropoly/sct_docker/blob/1e5b5261ebf9d2d0c3a26937a0318150abffb097/sct_docker.py#L252) for centos

Reply:
Yes, I can add it to the package needed by fsleyes.  I can check how to do it for other distros as well (use of `yum` for centos).